CHARTER.md

```diff
@@ -309,6 +310,56 @@ o. Any issues that cannot be resolved by the TOB shall be referred to The Linux
    Foundation Executive Director for mediation, and, if necessary, for
    resolution by The Linux Foundation Board of Directors.
 
+p. OCI Working Groups.
+
+   - i. The OCI may establish one or more working groups (each, an “OCI Working
+     Group”) to experiment, validate, and define new OCI Specifications or
+     major revisions to existing OCI Specifications. Each OCI Working Group
+     shall have a defined scope of the OCI Working Group's intended activities
+     and goals. The end result of the OCI Working Group's work product shall be
+     either:
+       - an update to an existing OCI Specification (for example, adding
+         support for a new set of features);
+       - a new OCI Specification (for example, creating an OCI Specification
+         for a new topical area); or
+       - a new OCI Project other than a Specification (for example, creating a
+         reference implementation of an OCI Specification).
+
+   - ii. Any participant in the OCI technical community may propose a working
+     group to the TOB for consideration. The TOB shall maintain a public list
+     of the information required to be included in a working group proposal.
+
+   - iii. Approval of a new OCI Working Group requires a two-thirds vote of the
+     TOB. The TOB will maintain a public list of all approved OCI Working
+     Groups.
+
+   - iv. Once an OCI Working Group determines that it has achieved its
+     objectives, the OCI Working Group leaders shall present to the TOB the
+     draft specification or other project, along with such other information as
+     the TOB may require for evaluation of the draft specification or other
+     project. The TOB shall maintain a public list of such required information.
+
+   - v. Approval of the draft specification (to be released as an OCI
+     Specification) or other draft project (to become an OCI Project) shall
+     require a two-thirds vote of the TOB. For a draft specification, prior to
+     such approval the OCI Working Group may not describe it as a released OCI
+     Specification, and must designate it with a `-draft.#` suffix.
+
+   - vi. Following TOB approval of a draft specification to be released as an
+     OCI Specification, the OCI Specification's maintainers may determine the
+     process for making subsequent releases, provided that (1) such process is
+     subject to TOB review, approval and modification, and (2) following any
+     release of a new version of an OCI Specification, OCI shall notify all
+     Members as set forth in Section 8(d) of this Charter.
+
+   - vii. The TOB may, by a two-thirds vote, approve requests from the owners
```

Comment:
nit: in `iv` above the term is "leaders" but her it's "owners"; we should use the same term in both places (whichever it is).